### PR TITLE
changes in timers, hall template given from backend

### PR DESCRIPTION
**Pull request by kds1660**

### CODE REVIEW

File: src/js/Controllers/hallCtrl.js [hunk 1]
```diff
@@ -1,76 +1,65 @@
...
+                    windowClass: 'modal',
+                    controller: 'LoginCtrl'
+                }).result.finally(function () {
+                    apiPost('users/login/social', JSON.stringify({
+                        "social_network": userService.getCurrentUser().network,
+                        "social_token": userService.getCurrentUser().accessToken
+                    })).then(function (response) {
+                        userService.addUser(response.data.user);
+                    }, function (error) {
+                        console.log(error);
```
Comment (Varkh) [inline src/js/Controllers/hallCtrl.js]: `console.error`

File: src/js/Controllers/hallCtrl.js [hunk 2]
```diff
@@ -79,41 +68,42 @@ angular.module('hall', ['ngFacebook','ui.bootstrap'])
...
             };
 
-            $scope.startTimer=function () {
-                if (cartService.getPlaceToCart().length&&!timerInterval) {
-                    endTime=new Date();
-                    endTime.setMinutes(endTime.getMinutes()+15);
+            $scope.startTimer = function () {
+                if (cartService.getPlaceToCart().length && !timerInterval) {
+                    endTime = new Date();
+                    endTime.setMinutes(endTime.getMinutes() + 15);
```
Comment (Varkh) [inline src/js/Controllers/hallCtrl.js]: What is 15 here?

File: app/views/Hall/Hall.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    </section>
+    <aside>
+        <aside-cart></aside-cart>
+    </aside>
+
+</main>
+
+<div ng-include="'views/shared/footer.html'"></div>
+
+<script>
```
Comment (Varkh) [inline app/views/Hall/Hall.html]: Maybe move to separate file?
Will it be used in other halls?

**Review by Varkh — CHANGES_REQUESTED** [review]

File: app/views/Hall/Hall.html (hunk 3 above)
Comment (kds1660) [reply app/views/Hall/Hall.html]: this is common file for all halls

File: src/js/Controllers/hallCtrl.js (hunk 2 above)
Comment (kds1660) [reply src/js/Controllers/hallCtrl.js]: customer asked to set 15min timer for buying tickets

File: src/js/Controllers/hallCtrl.js (hunk 1 above)
Comment (kds1660) [reply src/js/Controllers/hallCtrl.js]: changed

File: src/js/Controllers/hallCtrl.js (hunk 2 above)
Comment (kds1660) [reply src/js/Controllers/hallCtrl.js]: removed to timeForBuy variable

File: app/views/Hall/Hall.html (hunk 3 above)
Comment (kds1660) [reply app/views/Hall/Hall.html]: popover didn't work, i forget that receive hall from backend now and must call after response success, remove function in file and call it when template loading nowpopover didn't work, i forget that receive hall from backend now and must call after response success, remove function in file and call it when template loading now

**Review by spolischook — APPROVED** [review]

**Pull request merged**